web-services-app/src/main/java/edu/unc/lib/boxc/web/services/processing/IiifV3ManifestService.java

```diff
@@ -298,7 +299,8 @@ private boolean isVideo(String mimetype) {
     }
 
     private boolean isAudio(String mimetype) {
```

Comment:
I think at the moment we will only get audio IIIF manifests if the original file is mp4/mp3/m4a, but we can actually support all of the audio types that we permit m4a's be generated for. I think we need to update the logic of "isAudio" to check if the ContentObjectRecord has a audio access datastream and return true if so, in addition to checking the the mimetype for cases where we haven't generated the m4a yet.